cmd/vacuum/main.go

```diff
@@ -0,0 +1,21 @@
+// Copyright KubeArchive Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"flag"
+)
+
+func main() {
+	typePtr := flag.String("type", "namespace", "Type of vacuum, either 'namespace' or 'cluster'.")
+	configPtr := flag.String("config", "", "Name of configuration file.")
+
+	flag.Parse()
+
+	if *typePtr == "cluster" {
+		clusterVacuum(*configPtr)
+	} else {
+		namespaceVacuum(*configPtr)
```

Comment:
I think they should return an error and we should `os.Exit(1) on error, or something similar.